executor/executor_test.go

```diff
@@ -2341,9 +2341,9 @@ func (s *testSerialSuite) TestPointGetRepeatableRead(c *C) {
 		ctx = failpoint.WithHook(ctx, func(ctx context.Context, fpname string) bool {
 			return fpname == step1 || fpname == step2
 		})
-		rs, err := tk1.Se.Execute(ctx, "select c from point_get where b = 1")
+		rs, err := tk1.Se.ExecuteInternal(ctx, "select c from point_get where b = 1")
```

Comment:
I don't think we need to change it in the test cases. Can you get them back?

Reply:
This API is going to be removed.

@tiancaiamao suggested that we can have a wrapper around `Parse` and `ExecuteStmt`. Check [this](https://github.com/pingcap/tidb/pull/22652#issuecomment-772188830).

I think we should have an agreement on what we do with tests. We have #22647 from @AilinKid merged, which replaced `Execute` in `*_tests.go`.